web/src/js/components/Settings/SettingsPageComponent.js

```diff
@@ -120,11 +127,73 @@ class SettingsPage extends React.Component {
             justifyContent: 'center',
           }}
         >
-          {React.Children.map(this.props.children, child =>
-            React.cloneElement(child, {
-              showError: showError.bind(this),
-            })
-          )}
+          <Switch>
+            <Route
```

Comment:
Make this not exact and move to the bottom of the Switch so we fall back to this route for any /newtab/settings/* path.